server/utils/FirebaseAlgoliaServer/functions/tests/integrationTestHelpers.js

```diff
@@ -0,0 +1,311 @@
+// Initialize constants, libraries
+// These are public values 
+const config = {
+                  apiKey: "AIzaSyCR9KXK2iDuaZ3n6WNzZnBORbtpgWE7dvI",
+                  authDomain: "step9-2020-capstone.firebaseapp.com",
+                  projectId: "step9-2020-capstone"
+                };
+
+const admin = require('firebase-admin');
+admin.initializeApp(config);
+
+const db = admin.firestore();
+const COLLECTION = 'ads'; 
+
+const algoliasearch = require('algoliasearch');
+const functions = require('firebase-functions');
+
+const APP_ID = functions.config().algolia.app;
+const ADMIN_KEY = functions.config().algolia.key;
+
+// initialize algoliasearch API
+const client = algoliasearch(APP_ID, ADMIN_KEY);
+const index = client.initIndex('dev_ADS'); // current index
+
+const DEFAULT_SLEEP = 5000; // Firebase->Algolia takes a couple secs, WCS
```

Comment:
All this stuff should be set up in your test file and provided as parameters to methods in this file.  A helpers file is all static stuff.